src/sentry_plugins/jira/plugin.py

```diff
@@ -395,6 +395,11 @@ def create_issue(self, request, group, form_data, **kwargs):
                         v = [v]
                     elif schema.get('custom') == JIRA_CUSTOM_FIELD_TYPES.get('textarea'):
                         v = v
+                    elif schema['type'] == 'number':
+                        try:
+                            v = int(v)
```

Comment:
Just double check that this `number` value won't work with floats. If it does, this will truncate them.